src/modules/legacy/components/AccountsComparison/index.js

```diff
@@ -77,47 +75,45 @@ const AccountsComparison = ({ t }) => {
                 </>
               </Tooltip>
               <br />
-              {!hasEnoughBalance && (
```

Comment:
why this condition is removed?

Reply:
I think that logic was invalid, now we are checking for both balance and nonce, try to reclaim you see the user journey changes